### PR TITLE
Add blog post about the open build service connector

### PR DESCRIPTION
**Pull request by dcermak**
TODO:
- [x] Fix date once the text is final

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 504f91e9548fb484d7506e4557b5cde2b4e75b4a

https://deploy-preview-273--openbuildservice.netlify.app

File: _posts/development/2020-10-16-open-build-service-connector.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: An extension for Visual Studio Code
+category: development
+---
+
+As part of the developer engagement program, SUSE has kicked off the development of the [Open Build Service Connector](https://marketplace.visualstudio.com/items?itemName=SUSE.open-build-service-connector) an extension that brings the Open Build Service into Visual Studio Code!
+
+
+# Everything starts with a bookmark
```
Comment (dmarcoux) [inline _posts/development/2020-10-16-open-build-service-connector.md:10]: The headings should be properly capitalized. I often use https://capitalizemytitle.com/ to help me out with that I'm not unsure.

File: _posts/development/2020-10-16-open-build-service-connector.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: An extension for Visual Studio Code
```
Comment (dmarcoux) [inline _posts/development/2020-10-16-open-build-service-connector.md:3]: This is somewhat generic. I suggest going with something like `Open Build Service Connector` or `Introducing Open Build Service Connector`.

File: _posts/development/2020-10-16-open-build-service-connector.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+As part of the developer engagement program, SUSE has kicked off the development of the [Open Build Service Connector](https://marketplace.visualstudio.com/items?itemName=SUSE.open-build-service-connector) an extension that brings the Open Build Service into Visual Studio Code!
+
+
+# Everything starts with a bookmark
+
+The Open Build Service Connector is built around bookmarks of packages and projects. Bookmarks can be used to browse a project, its packages and their files. Additionally, you can view the configured repositories and adjust their project paths and architectures.
+
+<img src="/images/posts/2020-10-16/bookmarks.png" alt="View package files from bookmarks and the project's repositories" width="600px">
+
+Individual packages or whole projects can be checked out directly from within VSCode to the file system similarly as one would do via `osc`. OBS' version control is seamlessly integrated into Visual Studio Code's Source Control module and can be used in a comparable fashion to the git extension.
```
Comment (dmarcoux) [inline _posts/development/2020-10-16-open-build-service-connector.md:16]: We're referring to _Visual Studio Code_ as _Visual Studio Code_ except once, in this sentence, as `VSCode`. We should be consistent to not potentially confuse people.

File: _posts/development/2020-10-16-open-build-service-connector.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+<img src="/images/posts/2020-10-16/scm_quickdiff.png" alt="Changes to locally checked out packages are highlighted by VSCode" width="600px">
```
Comment (dmarcoux) [inline _posts/development/2020-10-16-open-build-service-connector.md:18]: Same as the comment above, either use _VSCode_ or _Visual Studio Code_.

**Comment by dcermak** [comment]
Thanks for taking a look @dmarcoux! I've tried to address your comments and pushed the changes.

**Comment by krauselukas** [comment]
@dcermak it looks pretty good to me, just remove the WIP and adjust the date, then this should be good to go :+1: very cool project by the way

File: _posts/development/2020-10-16-open-build-service-connector.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Introducing the Open Build Service Connector
+category: development
+---
+
+As part of the developer engagement program, SUSE has kicked off the development of the [Open Build Service Connector](https://marketplace.visualstudio.com/items?itemName=SUSE.open-build-service-connector) an extension that brings the Open Build Service into Visual Studio Code!
+
+
+# Everything starts with a bookmark
```
Comment (dmarcoux) [inline _posts/development/2020-10-16-open-build-service-connector.md:10]: The headings are not properly capitalized. This is how it should be for this one: 

```suggestion
# Everything Starts With a Bookmark
```

File: _posts/development/2020-10-16-open-build-service-connector.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Introducing the Open Build Service Connector
+category: development
+---
+
+As part of the developer engagement program, SUSE has kicked off the development of the [Open Build Service Connector](https://marketplace.visualstudio.com/items?itemName=SUSE.open-build-service-connector) an extension that brings the Open Build Service into Visual Studio Code!
```
Comment (dmarcoux) [inline _posts/development/2020-10-16-open-build-service-connector.md:7]:
```suggestion
As part of the developer engagement program, SUSE has kicked off the development of the [Open Build Service Connector](https://marketplace.visualstudio.com/items?itemName=SUSE.open-build-service-connector), an extension that brings the Open Build Service into Visual Studio Code!
```

**Comment by dcermak** [comment]
Lukas Krause <notifications@github.com> writes:

> @dcermak it looks pretty good to me, just remove the WIP and adjust the date, then this should be good to go :+1: very cool project by the way

Thanks! What should I set as the date? Tomorrow?

**Comment by dmarcoux** [comment]
> Thanks! What should I set as the date? Tomorrow?

Today is fine if this is merged today.

**Comment by dcermak** [comment]
@dmarcoux @krauselukas I've changed the date to today and hopefully fixed all the outstanding issues (the deployed version looks ok to me)

**Review by dmarcoux — APPROVED** [review]
Nice work @dcermak

**Review by krauselukas — APPROVED** [review]

**Comment by dcermak** [comment]
Thanks guys for merging this! Can I send around the announcement to the factory & obs mailinglists or is that automated as well?

**Comment by krauselukas** [comment]
@dcermak nope, you have to do it manually :)

**Pull request merged**